src/Dfc.CourseDirectory.WebV2/Features/Providers/EditProviderTypeConfirm.cshtml

```diff
@@ -0,0 +1,79 @@
+﻿@using Dfc.CourseDirectory.WebV2.Features.Providers.EditProviderType
+@model ConfirmViewModel
+@{ 
+    ViewContext.SetLayoutData(title: "Update provider type");
+
+    var affectedItemCountErrors = ViewData.ModelState[nameof(ConfirmViewModel.AffectedItemCounts)]?.Errors
+        ?? new Microsoft.AspNetCore.Mvc.ModelBinding.ModelErrorCollection();
+}
+
+<govuk-error-summary>
+    <govuk-error-summary-item asp-for="Confirm" data-testid="confirm-error" />
+    <govuk-error-summary-item asp-for="AffectedItemCounts" data-testid="affected-item-counts-error" />
+</govuk-error-summary>
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-heading-l">Update provider type</h1>
+
+        <p class="govuk-body">By updating the provider type, the following will be permanently deleted and no longer available on Publish to the course directory, Find a course or Tlevels.gov.uk.</p>
+
+        <form asp-action="Post" asp-all-route-data="@ProviderContext.RouteValues">
+
+            @Html.HiddenFor(m => m.ProviderId, new Dictionary<string, object> { { "data-testid", "provider-id" } })
```

Comment:
Not required.

Reply:
This is a deliberate change to prevent the possibility of accidently updating a different provider's provider type in the case where the provider context is coming from session and the provider context is changed to a different provider in a different window.